src/libs/pako.js

```diff
@@ -0,0 +1,7 @@
+var pako;
+
+(function() {
+  pako = require("pako");
+})();
+
+export { pako};
```

Comment:
I think we can just use `import pako from "pako"` directly in the filters file? AFAIK Rollup should be able to cope with that.